src/overmapbuffer.cpp

```diff
@@ -1251,7 +1251,7 @@ tripoint_abs_omt overmapbuffer::find_random(
     return find_random( origin, params );
 }
 
-shared_ptr_fast<npc> overmapbuffer::find_npc( character_id id )
+shared_ptr_fast<npc> overmapbuffer::find_npc( character_id id ) const
 {
     for( auto &it : overmaps ) {
```

Comment:
I assume you will fix these. Commenting so that it does not fly under the radar.